torch/nn/_functions/thnn/rnnFusedPointwise.py

```diff
@@ -10,37 +10,48 @@ def __init__(self):
     def forward(self, input_gate, hidden_gate, hx, ibias=None, hbias=None):
         if self.backend is None:
             self.backend = type2backend[type(input_gate)]
+
         hy = input_gate.new()
+        storage = input_gate.new().resize_(hx.numel() * 5)
```

Comment:
maybe assign it as `self.workspace` right here? It's not immediately clear what a "storage" is